CompilerSource/gcc_interface/gcc_backend.h

```diff
@@ -52,13 +53,13 @@ inline string fc(const char* fn) {
   }
 }
 
-inline int rdir_system(string x, string y) { return system((x + " " + y).c_str()); }
+inline int rdir_system(std::string x, std::string y) { return system((x + " " + y).c_str()); }
```

Comment:
Not a pressing issue, but honestly, `using std::string;` is pretty uncontroversial. You don't have to keep editing these headers to say `std::` everywhere, explicitly. It makes maintaining the code annoying.